src/aks-preview/azext_aks_preview/managed_cluster_decorator.py

```diff
@@ -2910,6 +2934,19 @@ def set_up_metrics_profile(self, mc: ManagedCluster) -> ManagedCluster:
 
         return mc
 
+    def set_up_ai_toolchain_operator_profile(self, mc: ManagedCluster) -> ManagedCluster:
+        self._ensure_mc(mc)
+
+        if self.context.get_ai_toolchain_operator() is not None:
```

Comment:
Default value of this option is `False`, not `None`, so even if user doesn't specify the option, the request body would have an empty `ManagedClusterAIToolchainOperatorProfile`

Reply:
Right, however in the `get_ai_toolchain_operator()` I may return None in few cases.